examples/oneapi/cpp/onedal_mac.lst

```diff
@@ -34,7 +34,6 @@ ONEDAL = column_accessor_homogen             \
          knn_cls_kd_tree_dense_batch         \
          knn_search_brute_force_dense_batch  \
          linear_kernel_dense_batch           \
-         load_graph                          \
```

Comment:
do we removing example from makefile? do not see new one being added

Reply:
there is no need to have an example on read function, any other graph example shows this functionality